src/main/java/frc/robot/commands/test/OdometryTestCommand.java

```diff
@@ -0,0 +1,58 @@
+package frc.robot.commands.test;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Transform2d;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Drivetrain;
+
+
+public class OdometryTestCommand extends CommandBase {
+  Drivetrain m_drive; 
+  Pose2d m_startPose;
+  double m_startTime;
+  Pose2d m_finalPose;
+  Transform2d m_distanceToMove;
+  Pose2d m_error;
+  
+  public OdometryTestCommand(Drivetrain drive, Transform2d distanceToMove) {
+    m_drive = drive; 
+    // finalPose is position after robot moves from current position-- startPose-- by the values that are inputted-- distanceToMove
+    m_distanceToMove = distanceToMove;
+    
+    addRequirements(drive);
+  }
+  
+  @Override
+  public void initialize() {
+    m_startTime = Timer.getFPGATimestamp();
+    m_startPose = m_drive.getPose();
+    m_finalPose = m_startPose.transformBy(m_distanceToMove);
+  }
+  
+  @Override
+  public void execute() {
+    m_drive.runChassisPID(m_finalPose.getX(), m_finalPose.getY(), m_finalPose.getRotation().getRadians()); 
+  }
+
+  @Override
+  public boolean isFinished() {
+    // TODO: the current PID values don't allow the command to finish
```

Comment:
Should we include a command that doesn't work?

Reply:
we can just increase the error margin and it will work but that is not the parameters we have on the design doc so I don't think it would be the best idea. I think it is fine to include the test and then have tuned PID constants in the next PR.